lib/creditsafe/request/find_company.rb

```diff
@@ -35,15 +46,29 @@ def message
         }
       end
 
-      private
+      # rubocop:disable Style/CyclomaticComplexity, Metrics/AbcSize
```

Comment:
As a programmer, I consider myself (and my colleagues) the ultimate consumers of a Rubocop like Metrics/AbcSize: it's ultimately designed to help reduce cognitive complexity and make things easier to understand. In this particular case, the criteria/error-message pairs are ... well, the various available criteria _are_ complex, but they're so arbitrary that I don't see any meaningful way to split them up that would actually improve understanding. So 👍  by me. Glad you've thought about it though :)